src/marqo/s2_inference/model_registry.py

```diff
@@ -1745,35 +23,9 @@ def _get_model_load_mappings() -> Dict:
             'hf':HF_MODEL}
 
 def load_model_properties() -> Dict:
-    # also truncate the name if not already
-    sbert_model_properties = _get_sbert_properties()
-    sbert_model_properties.update({k.split('/')[-1]:v for k,v in sbert_model_properties.items()})
-
-    sbert_onnx_model_properties = _get_sbert_onnx_properties()
-
-    clip_model_properties = _get_clip_properties()
-    test_model_properties = _get_sbert_test_properties()
-    random_model_properties = _get_random_properties()
-    hf_model_properties = _get_hf_properties()
-    open_clip_model_properties = _get_open_clip_properties()
-    onnx_clip_model_properties = _get_onnx_clip_properties()
-    multilingual_clip_model_properties = get_multilingual_clip_properties()
-    fp16_clip_model_properties = _get_fp16_clip_properties()
-
-    # combine the above dicts
-    model_properties = dict(clip_model_properties.items())
-    model_properties.update(sbert_model_properties)
-    model_properties.update(test_model_properties)
-    model_properties.update(sbert_onnx_model_properties)
-    model_properties.update(random_model_properties)
-    model_properties.update(hf_model_properties)
-    model_properties.update(open_clip_model_properties)
-    model_properties.update(onnx_clip_model_properties)
-    model_properties.update(multilingual_clip_model_properties)
-    model_properties.update(fp16_clip_model_properties)
 
     all_properties = dict()
-    all_properties['models'] = model_properties
+    all_properties['models'] = get_model_properties_dict()
```

Comment:
Interesting solution, to import the marqo commons registry into the marqo registry and keep the rest of the code mostly the same! The next steps would be:

1. Remove all model properties references from here and only return the loaders. Change function name to `get_model_loaders` and file name to `model_loaders.py`
2. Update all references to `load_model_properties` in the repo and replace them with references to marqo commons.

@pandu-k could I get feedback on this approach? Is it worth doing, or would it be better to keep things as is?